src/main/java/dk/sdu/se_f22/sharedlibrary/db/LoggingProvider.java

```diff
@@ -0,0 +1,81 @@
+package dk.sdu.se_f22.sharedlibrary.db;
+
+import org.apache.logging.log4j.*;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.core.appender.db.jdbc.*;
+
+public class LoggingProvider {
+    private static final Level logLevel = Level.ALL;
+
+    public static Logger getLogger(final Class<?> moduleName) {
+        ColumnConfig[] columnConfigs = getColumnConfigs();
+
+        // Sets up the appenders for the logging
+        var jdbcAppender = JdbcAppender.newBuilder()
+                .setName("DatabaseAppender")
+                .setConnectionSource(DBConnection.getInstance())
+                .setTableName("logs")
+                .setColumnConfigs(columnConfigs).build();
+        jdbcAppender.start();
+
+        // var consoleAppender = ConsoleAppender.newBuilder()
+        //         .setName("ConsoleAppender")
+        //         .setTarget(ConsoleAppender.Target.SYSTEM_OUT)
+        //         .build();
+        // consoleAppender.start();
```

Comment:
Is this going to be used? If not it should be removed, if it's going to be used it should be implemented.

Reply:
This should be fixed now.